src/services/expenses.service.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const { idGenerator } = require('./idGenerator');
+
+let expenses = [];
+
+const getAll = () => {
+  return expenses;
+};
```

Comment:
Your expenses are not working properly, you should return the data according to this URL. 
`https://localhost:3000/expenses?userId=4&categories=someCategory&from=11&to=12`

Pay attention to the request parameters.

Reply:
the request is incorrect you should use date but not numbers